src/7-to-8/major-changes/play-pause-stop.rst

```diff
@@ -29,9 +29,34 @@ commands:
 * ``cylc stop``
 
 A workflow can be safely played, paused and stopped any number of times without
-interrupting the workflow. The play command will always pick up where it left
-off. If you want to run again from the start, install a new run or start a
-:term:`reflow`.
+interrupting the workflow.
+
+
+Re-Running Workflows
+--------------------
+
+The ``cylc play`` command will always pick up where it left off (a
+:term:`restart`).
+
+If you want to re-run the entire workflow again from the start either:
+
+* Install a new run.
```

Comment:
Might be worth having a link to how to do this here. The suggestion is slightly generic, you might want to add a new reference at about line 145 of `user-guide/installing-workflows.rst` and link to that.

```suggestion
* :ref:`Install a new run<Using Cylc Install>`.
```